apps/trading/client-pages/market/trade-panels.tsx

```diff
@@ -66,6 +69,9 @@ export const TradePanels = ({
     <div className="h-full grid grid-rows-[min-content_1fr_min-content]">
       <div>
         {FLAGS.SUCCESSOR_MARKETS && <MarketSuccessorBanner market={market} />}
+        {FLAGS.SUCCESSOR_MARKETS && (
```

Comment:
chore (non blocking): wrap so we only need the condition once? (Same for trade-grid)

```
{FLAGS.SUCCESSOR_MARKETS && (
  <>
    <MarketSuccessorBanner market={market} />
    <MarketSuccessorProposalBanner marketId={market?.id} />
  </>
)}
```

Reply:
done